convert-hf-to-gguf.py

```diff
@@ -212,6 +212,7 @@ def from_model_architecture(cls, arch):
         try:
             return cls._model_classes[arch]
         except KeyError:
+            print(f"{cls._model_classes}")
```

Comment:
I don't think we need to leave this in. Unless you think the exception message should mention the full list of supported models.